Resources/Prototypes/Corvax/human_hair.yml

```
@@ -0,0 +1,569 @@
+﻿- type: marking
+  id: HumanHairAfricanPigtails
+  bodyPart: Hair
+  markingCategory: Hair
+  sprites:
+  - sprite: Corvax/Mobs/Customization/human_hair.rsi
```

yamllint

[error] wrong indentation: expected at least 3

(indentation)